docs/ReleaseSchedule.md

```diff
@@ -0,0 +1,17 @@
+# .NET MAUI Release Schedule Information
+
+Versions of .NET MAUI are being released in sync with new .NET versions. More information on the .NET release policy can be found [here](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
+
+## Past .NET MAUI Releases
+
+Below you can find a list of all the previous releases of .NET MAUI, excluding prereleases.
+For a full list, including release notes, please refer to our [Releases page](https://github.com/dotnet/maui/releases).
+
+| Version | Release Date | Days Since Last Release |
```

Comment:
Yeah I was on the fence about that one, but I guess the date will speak for itself. And apparently it's not as clear as I thought it would be :P

It's not something that counts daily, it simply is a Release Date minus Release Date from the previous line. So how many days there were between the release on a specific line and the one before that. Anyway, consider it removed!